graphql/schema.graphql

```diff
@@ -65,6 +72,7 @@ input SettingsInput {
   PerforceConnectionString: String
   PerforceUser: String
   PerforceTimeout: String
+  OverrideGlobal: Boolean
```

Comment:
Don't we want to only override some? let's say someone only wants to change the timeout?

Reply:
I read the rest and got what you were doing, but how do we know here what's overridden and what's global?

Reply:
The fields that are given are overriden, if for example `PerforceTimeout` wasn't supplied in the request, it wont be overriden

Reply:
It's a bit vague as this is a behavior expected from the FE and that might be broken one day by an unsuspecting noob. Comment? ❤️

Reply:
Let's discuss offline :)

Reply:
We just want to make sure the next dev understands that not giving a param means that this one is coming from the org